web/app/js/locales/en/messages.json

```diff
@@ -1,10 +1,24 @@
 {
   "404Msg": "Page not found.",
+  "A new version ({versionText}) is available.": "A new version ({versionText}) is available.",
   "Add {0} to the k8s.yml file<0/><1/>Then run {inject} to add it to the service mesh": "Add {0} to the k8s.yml file<0/><1/>Then run {inject} to add it to the service mesh",
   "All namespaces have a {productName} install.": "All namespaces have a {productName} install.",
+  "Control plane": "Control plane",
   "Control plane components": "Control plane components",
+  "Current {cmdNameDisplay} query": "Current {cmdNameDisplay} query",
   "Data plane proxies": "Data plane proxies",
+  "LinkerdIsUpToDateMsg": "Linkerd is up to date.",
+  "New service profile": "New service profile",
   "NoDataToDisplayMsg": "No data to display",
+  "Pod status: {0}": "Pod status: {0}",
+  "Service mesh details": "Service mesh details",
+  "Update Now": "Update Now",
```

Comment:
I think using the English phrase as key makes things easier, but it appears the original convention was to introduce camel-case keys such as `buttonCancel`, but this update isn't always following that convention?

Reply:
Good question, the camelCase keys such as `buttonCancel` are intended to be if the text is tied to a specific UI component, such as a button, form, etc., but there are instances of text throughout the dashboard that are just headlines/titles that are not tied to a specific display component or style, so I kept those as English phrases.